content/07-compiler-usage.md

```diff
@@ -178,3 +178,41 @@ Flag | Argument | Description | Platforms
 <!--include:generated/defines.md-->
 
 
+
+<!--label:compiler-usage-reporting-->
+### Error Reporting
+
+##### since Haxe 4.3.1
+
+The compiler can be configured to report errors and warnings in a format more readable on the command line. The define for this is `-D message.reporting=(mode)`, and the available modes are:
+
+* `classic` - the default, plain text format.
+* `indent` - similar to `classic`, but sub-messages (related errors or explanations of errors) are indented.
+* `pretty` - adds more contextual information to errors. For example, the expression or code fragment related to the error is displayed and highlighted with the error message.
```

Comment:
New in 4.3.1: `(macro)` is now added after severity for errors coming from macro context

Reply:
wait nvm it's not in 4.3.x